src/index.js

```diff
@@ -140,6 +140,14 @@ class TinderClient {
     }).then(response => response.data);
   }
 
+  getUpdates(fromTimestamp = '') {
```

Comment:
What do you think of naming the variable `since` vs. `fromTimestamp`? Or maybe `sinceTimestamp`?

It feels like `since` implies the concept of a datetime...but it might not imply the concept of a timestamp. I feel like it's consistent with how datetimes are represented by the client (i.e as strings), but I could see how it could be ambiguous.